pages/api/_utils.ts

```diff
@@ -0,0 +1,71 @@
+import { ConfigHelper, Config, LoggerInstance } from '@oceanprotocol/lib'
+import Web3 from 'web3'
+import ENS, { getEnsAddress as getEnsAddressVendor } from '@ensdomains/ensjs'
+import {
+  createClient,
+  dedupExchange,
+  TypedDocumentNode,
+  OperationContext,
+  fetchExchange
+} from 'urql'
+import { refocusExchange } from '@urql/exchange-refocus'
```

Comment:
This is a server, there is no user refocusing the browser window, so this is useless in this context